### PR TITLE
Fix layout of responses on narrow screens (e.g. mobile)

### LINKED ISSUE #175: Layout for responses is broken on mobile for long emails
Browser: Chrome 80
image

### PR DESCRIPTION
**Pull request by timjb**
**What changes does this PR introduce**

Fixes #175.

Video: https://drive.google.com/file/d/1EBT3tfd3s3VVnattb04RzKmULN6qqR4u/view

**Checklist**
- [x] `yarn build` passes
- [x] `yarn lint` does not show any errors

**Others details**
- [ ] This PR introduces a new dependency. Reason: _Mention your reason for introducing this dependency here_
  
If you have any questions be sure to let us know! For review ping anyone in this list:
 
 - @florianschmidt1994
 - @Maurice22

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -217,6 +217,7 @@ module.exports = {
     },
     flexGrow: {
       '0': '0',
+      '1000': '1000',
```
Comment (jagaSto) [inline tailwind.config.js]: Is there any specific reason we need to set the grow value to `1000`? Can it be anything arbitrary larger than `1`?

Comment (timjb) [reply tailwind.config.js]: Just tried this: It can be just one or any positive value whatsoever. Pushed a new commit where I simplified this.

**Comment by jagaSto** [comment]
Awesome, thanks for the changes and thank you for contributing @timjb ! This looks good from my side, as soon as we get another approval, we are ready to go 🚀

**Comment by florianschmidt1994** [comment]
Looks good, thanks for fixing it! I'm merging it now!

**Pull request merged**